demo/sdl_opengl3/main.c

```diff
@@ -70,7 +75,7 @@
  *                          DEMO
  *
  * ===============================================================*/
-int main(int argc, char *argv[])
+int main(void)
```

Comment:
Did you have any problems with these arguments since you removed them? I vaguely remember someone adding these arguments to fix some problem they had.

Reply:
That was accidental.  It was void up until 9 days ago and I started working on this long before that and just copied the version I had over after pulling.  I'll fix it.